src/Core/src/Handlers/TimePicker/TimePickerHandler.Android.cs

```diff
@@ -43,12 +44,23 @@ void onTimeSetCallback(object? obj, TimePickerDialog.TimeSetEventArgs args)
 
 				if (_dialog != null)
 				{
+					_dialog.DismissEvent -= dismiss;
```

Comment:
What would happen here if `dismiss` was `null`?

Reply:
@jonathanpeppers It is safe to unsubscribe from null, but just to be sure I checked it and nothing happened indeed